app/models/profile.rb

```diff
@@ -30,9 +30,9 @@ class Profile < ApplicationRecord
   validates :picture, content_type: ['image/png', 'image/jpeg']
 
   enum job_search_status: {
-    not_looking: 0,
+    not_job_searching: 0,
     open_to_opportunities: 1,
-    actively_looking: 2
+    open_to_work: 2
```

Comment:
Changed these because when actually viewing the profile, it felt strange to read things like "not actively looking". Not actively looking for what?

Assuming that people outside the agency may eventually see these pages, it's probably good to explicitly say things about "work", "job", etc. in these so it's very clear what's being communicated.

Also definitely don't consider what I have here the "best" and am interested on feedback. This naming is tricky to get right.